daemon/config.go

```diff
@@ -18,6 +18,11 @@ import (
 )
 
 const (
+	defaultMaxDownloadConcurrency = 5
+	defaultMaxUploadConcurrency   = 3
```

Comment:
Thanks @aaronlehmann that was a typo I must have been confused about which one is which when I was working on it.